mapie/classification.py

```diff
@@ -822,6 +815,8 @@ def predict(
         - Tuple[ArrayLike, ArrayLike] of shapes
         (n_samples,) and (n_samples, n_classes, n_alpha) if alpha is not None.
         """
+        if self.method == "top_k":
+            agg_scores = "mean"
```

Comment:
Why this choice ? It couldn't be "median" too ?

Reply:
No, only "mean" and "crossval" are valid choices. Using "median" does not give summed probabilities equal to 1.

Reply:
Ok, can we plan a flash meeting to discuss this point ?

Reply:
Okay !